x-pack/packages/kbn-elastic-assistant/impl/assistant/conversations/conversation_selector_settings/index.tsx

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiButtonIcon,
+  EuiComboBox,
+  EuiComboBoxOptionOption,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiHighlight,
+  EuiToolTip,
+} from '@elastic/eui';
+import React, { useCallback, useMemo, useState } from 'react';
+import { css } from '@emotion/react';
+
+import { OpenAiProviderType } from '@kbn/stack-connectors-plugin/common/gen_ai/constants';
```

Comment:
nit: consider `import type` and grouping it withe above imports
doesn't have to be in this PR